examples/cryptocurrency-advanced/backend/build.rs

```diff
@@ -1,10 +1,9 @@
 extern crate exonum_build;
 
 use exonum_build::protobuf_generate;
-use std::env;
 
 fn main() {
-    let exonum_protos = env::var("DEP_EXONUM_PROTOBUF_PROTOS").unwrap();
+    let exonum_protos = exonum_build::get_exonum_protobuf_files_path();
```

Comment:
It would be nicer to import `get_exonum_protobuf_files_files` along with `protobuf_generate`, just as you write in the rustdoc example.